LICENSE.txt

```diff
@@ -1,19 +1,34 @@
-Crate
-Copyright 2013-2016 Crate.IO GmbH ("Crate")
+CrateDB
+Copyright 2013-2017 Crate.io Inc. ("Crate.io")
```

Comment:
please take a look at the already approved template. i think the wording is "Crate.io Inc. and/or its affiliates"